requirements.txt

```diff
@@ -33,3 +33,4 @@ six
 sqlalchemy>=1.0.9
 stomp.py>=3.1.5
 suds-jurko>=0.6
+git+https://gitlab.cern.ch/fts/fts-rest.git@master
```

Comment:
I think this is not needed now (it is at line 2 here)

Reply:
Yep, removed